exporter/logzioexporter/to_domain.go

```diff
@@ -0,0 +1,255 @@
+// Copyright (c) 2019 The Jaeger Authors.
+// Copyright (c) 2018 Uber Technologies, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package logzioexporter
+
+import (
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/jaegertracing/jaeger/model"
+)
+
+// NewToDomain creates ToDomain
+func NewToDomain(tagDotReplacement string) ToDomain {
```

Comment:
where is this called from? I expect the exporter to be one way (to db model only), so I don't know why this one could be needed